spec/controllers/users_controller_spec.rb

```diff
@@ -7,7 +7,14 @@
     let(:attributes) { attributes_for :user }
 
     let :submit do
-      post :create, params: { user: attributes, roster_id: roster.id }
+      post :create, params: {
+        user: attributes.merge({ roster_ids: [roster.id] }) do |k, o, n|
+          next unless k == :roster_ids
+
+          o.concat(n)
+        end,
+        roster_id: roster.id
```

Comment:
`attributes_for :user` contains `roster_ids` already, so I had to get a little fancy here